timesketch/lib/analyzers/ssh_sessionizer_test.py

```diff
@@ -0,0 +1,175 @@
+"""Tests for SSHSessionizerSketchPlugin"""
+
+from __future__ import unicode_literals
+
+import mock
+
+from timesketch.lib.analyzers.ssh_sessionizer import SSHSessionizerSketchPlugin
+from timesketch.lib.testlib import BaseTest
+from timesketch.lib.testlib import MockDataStore
+from timesketch.lib.analyzers.sequence_sessionizer_test \
+    import _create_mock_event
```

Comment:
functions with a leading `_` are supposed to be private. Please change the name of the function if you want to import it here.